locales-en-US.xml

```diff
@@ -248,8 +269,8 @@
       <multiple>folios</multiple>
     </term>
     <term name="issue">
-      <single>number</single>
-      <multiple>numbers</multiple>
+      <single>issue</single>
+      <multiple>issues</multiple>
```

Comment:
@bwiernik Is this change too dangerous? There aren't many styles that use this, but surely one would more normally write out 'volume 1, issue 2' for 'vol. 1, no. 2'. It seems weird that we provide two different ways of getting the text 'number' but none for 'issue'.

Reply:
Yeah, I think this is reasonable. I rarely (if ever) see "number" spelled out when referring to issue number